test/fixtures/dialects/postgres/create_foreign_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE FOREIGN TABLE films (
```

Comment:
can you add some more test cases here? There's a lot of content added that isn't covered by these examples

Reply:
I've added some more examples in https://github.com/sqlfluff/sqlfluff/pull/5657/commits/a5a208a20ff306191811ed5e72c5a3c4cd1ca924 but, to be honest, I'm not very familiar with the syntax, which is why I only used the examples from the docs to begin with. 

I tested this update against the SQL file which motivated me to make this change in the first place and it passed but that was just a slightly more involved version of the first example, i.e. a relatively simple table definition but with column constraints.

I've done a bit of googling but I can't find anything that really goes beyond the basic examples.

So, what I'm testing here is two version of my reading of the grammar in the docs. If you can point me in the direction of some more complicated real world examples, I'd be happy to add some more tests!

Reply:
Ah, the column and table constraints are different from the normal CREATE TABLE ones. I'll have to change that...